src/sfntnames.c

```diff
@@ -129,7 +129,8 @@ Py_SfntNames_get_name(Py_SfntNames* self, PyObject* args, PyObject* kwds) {
         }
     }
 
-    Py_RETURN_NONE;
+    PyErr_Format(PyExc_KeyError, "No name of type %d", name);
+    return NULL;
```

Comment:
Learning the c-api very slowly.  https://docs.python.org/3.5/c-api/exceptions.html#c.PyErr_Format says it always returns `NULL`, could this be simplified to

```c++
return PyErr_Format(PyExc_KeyError, "No name of type %d", name);
```